roxctl/analyze/netpol/testdata/frontend-security/asset-cache-route.yaml

```diff
@@ -12,4 +12,5 @@ spec:
     weight: 100
   port:
     targetPort: http
-  wildcardPolicy: None
\ No newline at end of file
+  wildcardPolicy: None
+  
```

Comment:
There is a space here in line 16 - that is why it reports missing end line at the end of file. I recommend configuring your IDE to take care of such things automatically

Reply:
It applies also to some other yaml files - you can see them all in the Changes tab as Github highlights those issues.

Reply:
committed a fix , thanks